bottlecap/src/logs/flusher.rs

```diff
@@ -101,18 +102,21 @@ impl Flusher {
             .body(body)
     }
 
-    async fn send(req: reqwest::RequestBuilder) -> Result<(), Box<dyn Error>> {
+    async fn send(req: reqwest::RequestBuilder) -> Result<(), Box<dyn Error + Send>> {
         let time = Instant::now();
         let resp = req.send().await;
         let elapsed = time.elapsed();
+        debug!("Sent logs to datadog in {}ms", elapsed.as_millis());
 
         match resp {
             Ok(resp) => {
-                if resp.status() != 202 {
+                let status = resp.status();
+                _ = resp.text().await;
```

Comment:
Fixes an issue where keepalive can't be used if we don't fully read/exhaust the response buffer